terraform/terraform-modules/mwaa-env/variables.tf

```diff
@@ -0,0 +1,71 @@
+variable "airflow_env_name" {
+  description = "PDS Nucleus Airflow Env Name"
+  default     = "PDS-Nucleus-Airflow-Env"
+  type        = string
+}
+
+variable "airflow_version" {
+  description = "PDS Nucleus Airflow Version"
+  default     = "2.2.2"
+  type        = string
+}
+
+variable "airflow_env_class" {
+  description = "PDS Nucleus Airflow Environment Class"
+  default     = "mw1.medium"
+  type        = string
+}
+
+variable "airflow_dags_path" {
+  description = "PDS Nucleus Airflow DAGs Path"
+  default     = "dags/"
+  type        = string
+}
+
+variable "airflow_requirements_path" {
+  description = "PDS Nucleus Airflow Python Requirements File Path"
+  default     = "dags/"
+  type        = string
+}
+
+variable "region" {
+  description = "region"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "tags" {
+  description = "Default tags"
+  default     = {"env": "dev"}
+  type        = map(string)
+}
+
+variable "vpc_id" {
+  description = "VPC ID"
+  type        = string
+  default     = "<VPC ID>"
```

Comment:
That must be painful to update all these values. Should we store our terraform script in a private environment to avoid that ?